src/auth/factory.ts

```diff
@@ -0,0 +1,43 @@
+import { Factory } from 'src/common'
+import { UUIDs } from 'src/utils'
+import { User, UserStatus, UserTitle } from './user'
+
+export type UserFactoryParams = {
+  email: string
+  name: string
+  status?: UserStatus
+  title?: UserTitle
+  groupUuid: string
+}
+
+// TODO: Use Authgroup factory
+
+const defaultProps = {
+  hasProfilePicture: false,
+  status: UserStatus.INVITED,
+  title: undefined,
+  surveys: undefined,
```

Comment:
NEVER use undefined as default value

DO NOT ADD undefined or null default values